roles/DCOS.agent_windows/vars/main.yml

```diff
@@ -0,0 +1,13 @@
+# DCOS params:
+dcos_version_specifier: "{{ dcos['version'] }}"
+zookeeper: "{{ groups['masters'] | join(':2181,') }}:2181"
+
+# mesos-agent specific
+resources_ports: "{{ over_resources_ports | default('10000-34000') }}"
+mesos_agent_restart_count: 100
+mesos_agent_restart_interval: "PT1M" # every 1min
+system_user: "Administrator"
+
+# Packages location
+bucket_name: "dcos-win.s3.amazonaws.com"
+url_to_mesosbin: "http://{{ bucket_name }}/mesos-binaries.{{ dcos_version_specifier }}.zip"

```

Comment:
replace `dcos_version_specifier` with `dcos['version']`

Reply:
removed the variable `dcos_version_specifier` and using `{{ dcos['version'] }}` going forward